coffee_core/src/coffee.rs

```diff
@@ -545,12 +544,27 @@ impl PluginManager for CoffeeManager {
                     let mut actions = self.patch_repository_locally_absent(repos.to_vec()).await?;
                     nurse_actions.append(&mut actions);
                 }
+                Defect::CoffeeGlobalrepoCleanup(networks) => {
+                    let global_repo = format!("{}/repositories", self.config.root_path);
+                    for (network, path) in networks {
+                        log::info!("{network} - {path}");
+                        check_dir_or_make_if_missing(path.to_owned()).await?;
+                        if !Path::exists(Path::new(&path)) {
+                            copy_dir_if_exist(&global_repo, path).await?;
```

Comment:
`copy_dir_if_exist` doesn't return an error if the origin folder doesn't exist so if the global repositories folder doesn't exist, we wouldn't know.

Is this intended?